.circleci/config.yml

```diff
@@ -677,19 +680,35 @@ workflows:
   main-workflow:
     jobs:
       - build-job:
-          name: "Xcode_12.5.1_iOS_14.5"
+          name: "Xcode_12.5.1_iOS_14.5_Debug"
```

Comment:
Should we take this as an opportunity to start using Xcode `v13.x` for some build jobs?

Reply:
Also thought about it, I'll try updating to Xcode 13, if we won't need a lot of changes I'll make it in this PR